app/models/annotations/tag.rb

```diff
@@ -91,6 +91,19 @@ def hit_nested_objects_limit?
     ret
   end
 
+  def self.create_project_media_tags(project_media_id, tags_json)
+    project_media = ProjectMedia.find_by_id(project_media_id)
+
+    if !project_media.nil?
+      tags = JSON.parse(tags_json)
+      clean_tags = tags.map { |tag| tag.strip.gsub(/^#/, '') }.uniq
```

Comment:
I think we should also handle case sensitive to avoid duplication.
i.e  clean_tags = tags.map { |tag| tag.strip.gsub(/^#/, '') }.map(&:downcase).uniq